projects/packages/changelogger/src/Utils.php

```diff
@@ -187,9 +228,10 @@ public static function getTimestamp( $file, OutputInterface $output, DebugFormat
 	 * @param OutputInterface $output OutputInterface to write diagnostics too.
 	 * @param mixed           $files Output parameter. An array is written to this parameter, with
 	 *   keys being filenames in `$dir` and values being 0 for success, 1 for warnings, 2 for errors.
+	 * @param array|null      $input_options Holds command options retrieved from the InputInterface.
```

Comment:
Doesn't really matter where the options came from.
```suggestion
	 * @param array|null      $options Options for the extraction.
	 *   - add-pr-num: (bool) Whether to try to read a `(#12345)`-like PR number from the git commit creating each change entry. Default false.
```